ui-spacetimechart/src/assets/workScheduleSampleData.ts

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable import/no-unresolved */
+import {
+  type PathProperties,
+  type ProjectPathTrainResult,
+} from '@osrd-project/ui-manchette/dist/types';
+import { type WorkSchedule } from '@osrd-project/ui-spacetimechart/dist/types';
+
+export const SAMPLE_PATH_PROPERTIES_DATA: PathProperties = {
```

Comment:
I'd slightly prefer _not_ having this in `assets/` since it's not images/CSS/etc that need to be included in the final NPM package and it's only used in the story. But it seems like we already do the same thing for ui-manchette-with-spacetimechart. Feel free to resolve this comment.

Reply:
agreed, keeping that in the stories